shared/performance/performance.go

```diff
@@ -37,3 +39,14 @@ func CreatePerfTestCase(metricValue float32, metricName, testName string) junit.
 
 	return tc
 }
+
+// CreatePerfErrorRateTestCase creates a perf error rate test case with the provided value
+func CreatePerfErrorRateTestCase(metricValue float32, testName string) junit.TestCase {
```

Comment:
why are we creating an err property to the test case? We should treat it like a metric if the test wants to store this?

Reply:
I've removed this function, and changed the variable name to `perfPropertyName` to make it less confusing.
It seems unnecessary to change the actual name since the previous result will not be able to show correctly on testgrid.